k8s.gcr.io/k8s-staging-kube-state-metrics/manifest.yaml

```diff
@@ -0,0 +1,12 @@
+# google group for gcr.io/k8s-staging-kube-state-metrics is k8s-infra-staging-kube-state-metrics@kubernetes.io
```

Comment:
sorry for not catching this sooner...this needs to be `k8s-infra-staging-kube-metrics@kubernetes.io` too

Reply:
To be clear, I think the main problem is with the gcr repo name and not the google group name :grimacing:

Reply:
hmm .. that kind of messes with well known names of these projects, is there no other way? (the `k8s-staging-` prefix seems to take a pretty significant budget of that)

Reply:
cc @dims ^

Reply:
Yes this might cause confision, kube-metrics implies something completely different to me. I would suggest if there is no way around this, we can do `k8s-infra-staging-ksm`?